weather-app/src/app/day-forecast/day-forecast.component.html

```diff
@@ -0,0 +1,175 @@
+<div [ngClass]="{'dark-mode': isDarkMode}">
+    <div class="main-board">
+        <div class="title-table">
+            <span class="day-time">{{currentDay}}, {{currentTime}}</span>
+            <span class="location">{{currentCity}}, {{currentCountry}}</span>
+            <span><img src='../../assets/images/icons/star-dark-checked.svg' alt="star"></span>
+        </div>
+    <table>
+        <tr class="border">
+          <th class="border-right"></th>
+          <th class="border-right">0:00</th>
```

Comment:
I'd suggest to move as many weather parameters as possible to mocked variables in .ts (arrays of values for example)
To display them here with ngFor.

Reply:
Fixed.